content/blog/2018/08/2019-08-31-speaker-blog-delivery-automation.adoc

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "Cultural Transformation via Delivery Automation"
+tags:
+- jenkins

```

Comment:
YAGNI. Surprisingly, everything on this site is about Jenkins. Note that there is also nothing Jenkins-specific in the blogpost excepting the name reference in the first sentence